File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int n = nums.size();
        unordered_map<int,int> map;

        for(int el : nums){
            if(!map.count(el)){
                map[el]++;
            }
            else{
                return true;
            }
        }

        return false;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        my_dict = defaultdict(int);
        for num in nums:
            my_dict[num]+=1
        
        my_keys = sorted(my_dict, key=my_dict.get, reverse=True)
        ans =[]
        for i in range(k):
            ans.append(my_keys[i])
        return ans